dist/eModal.js

```diff
@@ -231,12 +231,12 @@
 
             function createModalElement() {
                 return $('<div class="modal fade" tabindex="-1"><style>.modal-xl{width:96%;}.modal-body{max-height: calc(100vh - 145px);overflow-y: auto;}</style>' +
-                    '<div class=modal-dialog>' +
-                    '<div class=modal-content>' +
-                    ' <div class=modal-header><button type=button class="x close" data-dismiss=modal><span aria-hidden=true>&times;</span><span class=sr-only>Close</span></button><h4 class=modal-title></h4></div>' +
-                    '</div>' +
-                    '</div>' +
-                    '</div>')
+                        '<div class=modal-dialog>' +
```

Comment:
This is just creating noise in the diff, can we please revert the tab situation!